patches/chromium/fix_implement_print_preview_methods_in_printviewmanagerbasic.patch

```diff
@@ -0,0 +1,76 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Mon, 17 May 2021 14:47:35 +0200
+Subject: fix: implement Print Preview methods in PrintViewManagerBasic
+
+Print Preview code was removed from PrintManager in https://chromium-review.googlesource.com/c/chromium/src/+/2888590
+and the removed methods were implemented in HeadlessPrintManager but
+not PrintViewManagerBasic, where they are also required. This caused
+compilation errors in Electron when we pulled in the relevant CL,
+as we make use of PrintViewManagerBasic.
+
+This fixes the issue by implementing the missing methods in
+PrintViewManagerBasic.
+
+Upstreamed at https://chromium-review.googlesource.com/c/chromium/src/+/2897572
```

Comment:
The comment here is:

> This shouldn't be necessary.

thoughts?

Reply:
@nornagon i'm still talking to him via Slack but Chromium's config differs from ours - we use PrintViewManagerBasic _for_ print preview, which they explicitly don't, so while this shouldn't be necessary in a we-shouldn't-be-doing-this-at-all kind of way, pending a fairly large refactor we do need to do this (also i removed that patch - the only one there now should be a revert of his)

Reply:
Updated this per commentary on the CL  - removed the patch and subclasses to `PrintViewManagerElectron`